internal/node/node.go

```diff
@@ -13,6 +14,9 @@ type Node interface {
 	GetNodesListBySelector(ctx context.Context, selector map[string]string) ([]v1.Node, error)
 	GetNumTargetedNodes(ctx context.Context, selector map[string]string) (int, error)
 	FindNodeCondition(cond []v1.NodeCondition, conditionType v1.NodeConditionType) *v1.NodeCondition
+	UpdateLabels(ctx context.Context, node *v1.Node, toBeAdded, toBeRemoved []string) error
+	addLabels(node *v1.Node, labels []string)
+	removeLabels(node *v1.Node, labels []string)
```

Comment:
Let's remove them from the interface and make then non-exported *functions*.

https://github.com/kubernetes-sigs/kernel-module-management/pull/850#discussion_r1673821877